report_label/models/report_paperformat_label.py

```diff
@@ -3,9 +3,10 @@
 
 class ReportPaperformatLabel(models.Model):
     _name = "report.paperformat.label"
-    _inherits = {"report.paperformat": "paperformat_id"}
```

Comment:
I wonder what's the motivation behind this change?

IMO this inheritance made perfect sense, no?

In fact this would be the place to set a default value for the `disable_shrinking` field as mentioned above, if the inheritance is kept

Reply:
agree with @ivantodorovich , 
maybe we're missing the benefits of the non inheritance but keeping seems to be a better option IMHO.
Could you explain what was your idea?
Regards

Reply:
Basically, to have the possibility to use the same report.paperformat for many report.paperformat.label.
In the current design, when you create a report.paperformat.label, it creates automatically a report.paperformat, even if one exists and is OK.
configuring report.paperformat is one thing, configuring report.paperformat.label is another one. (even if a report.paperformat is required when we have to create report.paperformat.label).

I tested and it doesnt seems to break anything.

Reply:
Well, in my mind, each label kind **is** a paperformat. Only it's not a regular paperformat, it's a "label" paperformat, that contains extra information.. thus the inheritance.

So when you say the label paperformat  should be able to re-use existing paperformats, I find it a bit weird, but I see what you mean.

In any case, I'm not against it